docs/docs/cli/installation.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Installation
+sidebar_label: Installation
+hide_table_of_contents: false
+---
+
+import Highlighter from "@site/src/components/Highlighter";
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Highlighter>
+Install the CLI quickly with our step-by-step guide or build it from source
+</Highlighter>
+
+## Ways to Install
+
+Installing the Airy CLI is easy and straightforward.
+You can follow the next steps for a quick setup:
+
+- [Step 1: Check the requirements](installation.md#step-1-check-the-requirements)
+- [Step 2: Install Airy CLI](installation.md#step-2-install-the-airy-cli)
+- [Step 3: Verify your installation](installation.md#step-3-verify-your-installation)
+- [Step 4: Initialize the setup](installation.md#step-4-initialize-the-setup)
+
+
+
+### Step 1: Check the requirements
+
+Airy is built to run in the cloud, but you can also run it locally.
+The CLI runs on macOS, Linux and Windows and we provide prebuildt executables for **x86-64**.
+
+If you are running on a different platform you can also [build it yourself](installation.md#building-from-source)
+
+
+### Step 2: Install the Airy CLI
+
+
+#### Download the latest release with:
+
+<Tabs
+groupId="operating-systems"
+defaultValue="mac"
+values={[
+{label: 'macOS', value: 'mac'},
+{label: 'Linux', value: 'linux'},
+]
+}>
+<TabItem value="mac">
+
+```bash
+curl "https://airy-core-binaries.s3.amazonaws.com/$(curl -L -s https://airy-core-binaries.s3.amazonaws.com/stable.txt)/darwin/amd64/airy" -o "airy"
+```
+
+:::note
+
+To download a specific version, replace the `$(curl -L -s <URL>)` portion of the command with the specific version.
+
+For example, to download version 0.6.0 on macOS, type:
+
+curl https://airy-core-binaries.s3.amazonaws.com/0.6.0/darwin/amd64/airy -o "airy"
+:::
+</TabItem>
+
+<TabItem value="linux">
+
+```bash
+curl -LO "https://airy-core-binaries.s3.amazonaws.com/$(curl -L -s https://airy-core-binaries.s3.amazonaws.com/stable.txt)/linux/amd64/airy" -o "airy"
+```
+
+:::note
+
+To download a specific version, replace the `$(curl -L -s <URL>)` portion of the command with the specific version.
+
+For example, to download version 0.6.0 on Linux, type:
+
+curl https://airy-core-binaries.s3.amazonaws.com/0.6.0/linux/amd64/airy -o "airy"
+:::
+</TabItem>
+</Tabs>
+
+
+<Tabs
+groupId="operating-systems"
+defaultValue="mac"
+values={[
+{label: 'macOS', value: 'mac'},
+{label: 'Linux', value: 'linux'},
+]
+}>
+
+<TabItem value="mac">
+
+
+#### Make it executable
+
+```bash
+chmod +x ./airy
```

Comment:
I would merge the ke it executable with the previous step. It's too small to deserve its own step?